ultranest/popstepsampler.py

```diff
@@ -536,8 +536,219 @@ def __next__(
         else:
             return None, None, None, nc
 
+class PopulationSimpleSliceSampler():
+    """
+       Vectorized Slice sampler without stepping out procedure.
```

Comment:
Vectorized Slice sampler without stepping out procedure for quick look fits.

Reply:
Added